src/v2/Apps/Components/AppShell.tsx

```diff
@@ -75,48 +77,58 @@ export const AppShell: React.FC<AppShellProps> = props => {
   useNetworkOfflineMonitor()
   useProductionEnvironmentWarning()
 
+  const localizations = {
+    en_US: locale_en_us,
+  }
+
   return (
-    <Flex
-      width="100%"
-      // Prevents horizontal scrollbars from `FullBleed` + persistent vertical scrollbars
-      overflowX="hidden"
-      // Implements "Sticky footer" pattern
-      // https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/
-      minHeight="100vh"
-      flexDirection="column"
+    <IntlProvider
+      messages={localizations["en_US"]}
+      locale="en-us"
+      defaultLocale="en-us"
```

Comment:
We should put this provider in `Boot` with the other providers.

Reply:
We can move this to boot, but better would be to write our own component that wraps this and encapsulates the other setup functionality in one location, eg 

 ```tsx
 // v2/System/i18n/InternationalizationProvider.tsx 
 
import { IntlProvider } from "react-intl"
import locale_en_us from "../../compiled-lang/en-us.json"
 
 const localizations = {
    en_US: locale_en_us,
  }
  
export const InternationalizationProvider: React.FC = ({ children }) => {
  return (
    <IntlProvider
      messages={localizations["en_US"]}
      locale="en-us"
      defaultLocale="en-us"
    >
      {children}
    </IntlProvider>
  )
}
 ```
 
 And then import this into `Boot` as suggested above.

Reply:
Also, note the output of the json file above: `__i18n__` -- this matches idioms for our other compiled target output `v2/__generated__`, for relay.